Core/Utilities/ElasticSearchDB.py

```diff
@@ -0,0 +1,272 @@
+########################################################################
+# $Id: $
+########################################################################
+
+"""
+This class a wrapper around elasticsearch-py. It is used to query
+Elasticsearch database.
+
+"""
+
+__RCSID__ = "$Id$"
+
+from DIRAC                      import gLogger, S_OK, S_ERROR
+from elasticsearch              import Elasticsearch
+from elasticsearch_dsl          import Search, Q, A
+from elasticsearch.exceptions   import ConnectionError, TransportError, NotFoundError
+from elasticsearch.helpers      import BulkIndexError
+from elasticsearch              import helpers
+from datetime                   import datetime
+from DIRAC.Core.Utilities       import Time
```

Comment:
I want to get the epoch. How do you want to get the epoch? Could you please give me an example? 
Do You know int(time.time()) != Time.toEpoch()?
I am not favour for dropping the time module....